dateconv/dateconv.go

```diff
@@ -72,6 +73,22 @@ func BsDaysInMonthsByYear(yy int, mm time.Month) (int, bool) {
 	return months[query], true
 }
 
+// TotalDaysInBSYear returns total number of days in a particular BS year.
+func TotalDaysInBSYear(bsYear int) (int, error) {
+	sum := 0
```

Comment:
Please initialize this right above the `for` loop. This feels very C-like to me 😄